flang/lib/Lower/OpenMP.cpp

```diff
@@ -103,7 +103,12 @@ genOMP(Fortran::lower::AbstractConverter &converter,
             genOMP(converter, eval, simpleStandaloneConstruct);
           },
           [&](const Fortran::parser::OpenMPFlushConstruct &flushConstruct) {
-            TODO("");
+            // TODO: Handle the clause associated with Flush
```

Comment:
Can this continue to be a hard TODO() with some sort of condition testing? (Hard TODOs are greatly preferred to comments.)

Reply:
It will be un-conditional if we go that way(since clauses will not survive the semantic phase). How about a FIXME ?

Reply:
The comment is ok as is if you don't see an alternative. (Just checking.)